docs/lsp-clients.json

```diff
@@ -243,6 +243,23 @@
     "installation-url": "https://github.com/emacs-lsp/lsp-mode/wiki/Install-EmmyLua-Language-server",
     "debugger": "Not available"
   },
+  {
+    "name": "lua",
+    "full-name": "Lua",
+    "server-name": "lua-language-server",
+    "installation-url": "https://github.com/sumneko/lua-language-server/wiki/Build-and-Run-(Standalone)",
+    "server-url": "https://github.com/sumneko/lua-language-server",
+    "debugger": "Not available"
+  },
+  {
+    "name": "lua",
```

Comment:
This name must be unique, could you rename these names to some names that don't conflict?

Reply:
done